blogposts/2022/sourcegraph-non-devs.md

```diff
@@ -0,0 +1,207 @@
+---
+title: 'Sourcegraph for non-devs'
+description: "Sourcegraph isn't just for software developers. Learn how product managers and analytics/business operations teammates use Sourcegraph as a powerful timesaver and cheat sheet that saves them 20-60 minutes multiple times per week."
+author: Eric Brody-Moore, Joel Kwartler
+publishDate: 2022-02-17T00:00-07:00
+tags: [blog]
+slug: sourcegraph-for-non-devs
+heroImage:
+socialImage:
+published: true
+---
+
+Sourcegraph is an extremely powerful developer tool used by engineers at top companies, from Uber and Lyft to Amazon and SoFi. But we’ve also discovered it’s a powerful timesaver and cheat sheet for teams across an organization, such as Product Management, Business Operations, Analytics, and Marketing.
```

Comment:
This reads a little too heavy on marketing for me. I don't want to take words out of the mouths of the authors, but maybe we could soften the promotion here?